server/game/cards/02_SHD/units/RicketyQuadjumper.ts

```diff
@@ -0,0 +1,33 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+
+export default class RicketyQuadjumper extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '7291903225',
+            internalName: 'rickety-quadjumper'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addOnAttackAbility({
+            title: 'Reveal a card',
+            optional: true,
+            immediateEffect: AbilityHelper.immediateEffects.reveal((context) => ({ target: context.source.controller.getTopCardOfDeck() })),
+
+            ifYouDo: (context) => ({
```

Comment:
nit: we typically call this variable `ifYouDoContext` to distinguish from any other context that might be used deeper inside